witnet.toml

```diff
@@ -14,10 +14,10 @@ enabled = true
 server_address = "127.0.0.1:21338"
 
 [consensus_constants]
-checkpoints_period = 30
-checkpoint_zero_timestamp = 1584111600
-bootstrap_hash = "00000000000000000000000000000000000000007769746e65742d302e372e30"
-genesis_hash = "7a7a438b857e8a50fb3281df32b34f8acd930da8f0d51f308903c8a55e6ff7a9"
+checkpoints_period = 45
```

Comment:
When was this discussed?

Reply:
During last consensus constants assessment session.